### PR TITLE
Add TrustedTeamsBlock and assets for it

### PR DESCRIPTION
**Pull request by Kragov**

### CODE REVIEW

File: docs/src/components/landingBlocks/TrustedTeamsBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  title: 'Trusted by top teams',
+};
+
+type Props = {
+  content: typeof content;
+};
+
+const { media } = appTheme;
+
+const Section = styled.section`
```
Comment (usulpro) [inline docs/src/components/landingBlocks/TrustedTeamsBlock.tsx]: Same here - can't we reuse the existing Section with fullwidth prop?

File: docs/src/components/landingBlocks/TrustedTeamsBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+      transform: translateX(0);
+    }
+
+    100% {
+      transform: translateX(-100%);
+    }
+  }
+`;
+
+const logosRowsCreator = (rowsAmount: number) => {
```
Comment (usulpro) [inline docs/src/components/landingBlocks/TrustedTeamsBlock.tsx]: function name should use verbs e.g. `createLogosRows`

**Comment by usulpro** [comment]
image
on production rows direction is another and speeds aren't equal

File: docs/src/components/landingBlocks/TrustedTeamsBlock.tsx [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  const logosInRowAmount = Math.floor(TrustedTeamsLogos.length / rowsAmount);
+  const logosRows = [];
+
+  for (let i = 0; i < rowsAmount; i++) {
```
Comment (usulpro) [inline docs/src/components/landingBlocks/TrustedTeamsBlock.tsx]: why not array.map here?

**Comment by usulpro** [comment]
missed padding 
image

**Comment by usulpro** [comment]
image
Images looks too big on mobile

File: docs/src/assets/images/trustedTeamsLogos/index.ts [hunk 4]
```diff
@@ -0,0 +1,93 @@
+import LodestarImage from './lodestar.png';
```
Comment (usulpro) [inline docs/src/assets/images/trustedTeamsLogos/index.ts]: let's move this file from assets to somewhere closer to component

Comment (usulpro) [reply docs/src/assets/images/trustedTeamsLogos/index.ts]: e.g. it could be `docs/src/components/landingBlocks/TrustedTeamsBlock.model.tsx`

**Review by usulpro — CHANGES_REQUESTED** [review]
also add storybook, check lintings and preview

File: docs/src/components/landingBlocks/TrustedTeamsBlock.tsx (hunk 3 above)
Comment (Kragov) [reply docs/src/components/landingBlocks/TrustedTeamsBlock.tsx]: There are no array that can be mapped to get necessary result

File: docs/src/components/landingBlocks/TrustedTeamsBlock.tsx (hunk 2 above)
Comment (Kragov) [reply docs/src/components/landingBlocks/TrustedTeamsBlock.tsx]: Done

File: docs/src/components/landingBlocks/TrustedTeamsBlock.tsx (hunk 1 above)
Comment (Kragov) [reply docs/src/components/landingBlocks/TrustedTeamsBlock.tsx]: Done

File: docs/src/assets/images/trustedTeamsLogos/index.ts (hunk 4 above)
Comment (Kragov) [reply docs/src/assets/images/trustedTeamsLogos/index.ts]: Done

**Pull request merged**